tutorials/katacoda/thanos/2-lts/index.json

```diff
@@ -1,17 +1,48 @@
 {
   "title": "Intro: Downsampling and unlimited metric retention for Prometheus",
```

Comment:
Is Downsampling part of it? I think we talked about fetching data from storage, that's it (:

Reply:
Right, here, we won't be showing this. Instead, we will be adding compaction to show retention capabilities and as far discussed in our 1:1 meeting :)

Reply:
Right, here, we won't be showing this. Instead, we will be adding compaction to show retention capabilities and as far discussed in our 1:1 meeting :)